constellation-app/constellation
# Refactor info map options dialog box

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change


Refactor info map dialog box to have styling consistent with the rest of consty. Uses the `PluginParametersSwingDialog` code instead of swing dialog.
Also, the original issue suggests changing the "Connection Type" and "Dynamic Type" parameters be changed to checkboxes as multiple options can be selected at once. From what I understand this wasn't intentional, so I kept these as single choice parameters.

### Alternate Designs


N/A

### Why Should This Be In Core?


N/A

### Benefits


Makes the dialog box style more consistent with the rest of consty.

### Possible Drawbacks


None

### Verification Process



1. Open new graph
2. Tools -> Cluster -> Info Map...
3. Observe styling
4. Use info map with different configurations and observe same results as info map from master branch

### Applicable Issues


https://github.com/constellation-app/constellation/issues/2119
